articles/configure-a-404-page.mdx

```diff
@@ -0,0 +1,103 @@
+By default, visitors will see a simple 404 page when their browser cannot find a URL from a Site hosted on CloudCannon.
+
+<comp.DocsImage path="https://cc-dam.imgix.net/documentation/images/hosting/configure-a-404-page/2025-01/CloudCannon-Documentation-404-Page.png" alt="A screenshot of CloudCannon's default 404 page." title="CloudCannon's Default 404 Page" type="screenshot"/>
+
+We recommend creating a dedicated 404 page that matches your Site branding and provides helpful resources when your users get lost.
+
+To redirect visitors to a custom 404 page:
+
+1. Create a file called `404.html` and add the content you want CloudCannon to serve.
+2. Save your page to your Site. We recommend saving it in the same place as other important pages (e.g., "Home" or "About").
+3. Navigate to the `routing.json` file within your `.cloudcannon` folder using the *File browser*, or create one if you have not done so.
+4. Create an entry within the `routes` array with the subkeys `from`, `to`, `status`, and `forced`.
+5. Set `from` to a REGEX string that captures all URLs (e.g., `/.*`), `to` to the path of your 404 page, `status` to `404`, and `forced` to `false`.
+6. Save your changes.
+
+After a successful build, CloudCannon will redirect visitors to `404.html` when their browser cannot find a page on your website.
+
+It is important to set `forced` to `false`, as this configuration option determines whether the redirect rule should take priority over any existing page content. By setting this key to `false`, CloudCannon will only serve our 404 page content for URLs that match the `from` REGEX string if the browser cannot find a page's content.
+
+For a complete list of options available in the `routing.json` file, please read our [routing reference](/documentation/articles/routing-reference/) documentation.
+
+<comp.Notice info_type="info">
+  Set the permalink of your page to /404.html if you are using Jekyll and non-default permalinks.
+</comp.Notice>
+
+If you would prefer, you can also configure your `routing.json` file using your local development environment.
+
+Here is an example of what your 404 redirect should look like:
+
+<comp.CodeBlock language="json" source="routing.json">
+```
+
+{
+  "routes": [
+    {
+      "from": "/.*",
+      "to": "/404.html",
+      "status": 404,
+      "forced": false
+    }
+  ]
+}
+
+```
+</comp.CodeBlock>
+
+Most Sites only need one 404 page, but you can create as many 404 redirects as required by configuring a more specific `from` URL. For more generic information, please read our documentation on [configuring custom routing](/documentation/articles/configure-custom-routing/).
+
+<comp.CodeBlock language="json" source="routing.json">
+```
+
+{
+  "routes": [
+    {
+      "from": "/.*",
+      "to": "/404.html",
+      "status": 404,
+      "forced": false
+    },
+    {
+      "from": "/blog/.*",
+      "to": "/404-blog.html",
+      "status": 404,
+      "forced": false
+    }
+  ]
+}
+
+```
+</comp.CodeBlock>
```

Comment:
Have you tested this one? I didn't think these were resolved on specificity, I believe they're list order (so the top rule in this case would always match, but it would work as intended if the order was reversed).